pkgs/games/steam/runtime.nix

```diff
@@ -35,7 +33,7 @@ in stdenv.mkDerivation rec {
     description = "The official runtime used by Steam";
     homepage = https://github.com/ValveSoftware/steam-runtime;
     license = licenses.mit;
-    maintainers = with maintainers; [ hrdinka ];
+    maintainers = with maintainers; [ hrdinka abbradar ];
     hydraPlatforms = [];
```

Comment:
Why? Is the output very large? Maybe it would be worth it to get `"x86_64-linux"` binaries from Hydra for this.

On second thought, download time and potential disk overhead (before GC) is probably of little concern, as steam games typically need ample amount of both.